packages/flutter_tools/lib/src/test/test_compiler.dart

```diff
@@ -81,6 +86,14 @@ class TestCompiler {
       await compiler.shutdown();
       compiler = null;
     }
+
+    // If we created a copy of the test dill file for this process only, clean it up.
+    if (testWithoutLockingFilePath != null) {
+      final File testFile = fs.file(testWithoutLockingFilePath);
+      if (await testFile.exists()) {
```

Comment:
Prefer to use `existsSync()`and `deleteSync()`.

Reply:
Done.